DQMOffline/Muon/src/GEMEfficiencyAnalyzer.cc

```diff
@@ -6,13 +6,17 @@
 #include "DataFormats/Math/interface/deltaPhi.h"
 #include "DataFormats/GeometryCommonDetAlgo/interface/ErrorFrameTransformer.h"
 #include "DataFormats/GeometrySurface/interface/SimpleDiskBounds.h"
-#include "Geometry/CommonTopologies/interface/GlobalTrackingGeometry.h"
-#include "Geometry/Records/interface/GlobalTrackingGeometryRecord.h"
 #include "Geometry/CommonTopologies/interface/StripTopology.h"
 #include "Validation/MuonHits/interface/MuonHitHelper.h"
 #include "Validation/MuonGEMHits/interface/GEMValidationUtils.h"
 
-GEMEfficiencyAnalyzer::GEMEfficiencyAnalyzer(const edm::ParameterSet& pset) : GEMOfflineDQMBase(pset) {
+GEMEfficiencyAnalyzer::GEMEfficiencyAnalyzer(const edm::ParameterSet& pset)
+    : GEMOfflineDQMBase(pset),
+      gemToken1_(esConsumes<GEMGeometry, MuonGeometryRecord>()),                                                                    
```

Comment:
This needs to be - gemToken1_(esConsumes<edm::Transition::BeginRun>())

Reply:
thanks, I have updated it.